engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -4731,7 +4732,8 @@ protected VirtualMachine retrieve() {
         }
         catch (CloudRuntimeException e){
             s_logger.info("Caught CloudRuntimeException, returning job failed");
```

Comment:
@niteshsarda can you at least log the caught error here. I'm fine with throwing/returning a general error message to the user.

Reply:
@rhtyd : Changed code as per suggestion.